Tests fail when using Go version 1.10.x

error:

```
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v  -timeout 120m
?   	github.com/articulate/terraform-provider-okta	[no test files]
# github.com/articulate/terraform-provider-okta/okta
okta/provider_test.go:50: Fatal call has possible formatting directive %v
FAIL	github.com/articulate/terraform-provider-okta/okta [build failed]
make: *** [testacc] Error 2
```
temporary fix is to force the tests phase to use golang version 1.9.x

will need to circle back and make changes to support version 1.10.x

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -12,7 +12,9 @@ build: fmtcheck
 	go build
 
 test: fmtcheck
-	go test $(TEST) -timeout=30s -parallel=4
+	go test -i $(TEST) || exit 1
+	echo $(TEST) | \
+		xargs -testacc -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
 testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -14,7 +14,7 @@ build: fmtcheck
 test: fmtcheck
 	go test -i $(TEST) || exit 1
 	echo $(TEST) | \
-		xargs -testacc -n4 go test $(TESTARGS) -timeout=30s -parallel=4
+		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
 testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -3,7 +3,7 @@ GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 
 default: deps build
 
-deps: 
+deps:
 	curl -s https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 	${GOPATH}/bin/dep ensure
 	${GOPATH}/bin/dep ensure -update github.com/articulate/oktasdk-go
@@ -12,9 +12,7 @@ build: fmtcheck
 	go build
 
 test: fmtcheck
-	go test -i $(TEST) || exit 1
-	echo $(TEST) | \
-		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
+	go test $(TEST) -timeout=30s -parallel=4
 
 testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m

--- a/okta/provider_test.go
+++ b/okta/provider_test.go
@@ -40,19 +40,19 @@ func testAccPreCheck(t *testing.T) {
 }
 
 func testOktaConfig(t *testing.T) *Config {
-	testAccPreCheck(t)
 	config := Config{
 		orgName:  os.Getenv("OKTA_ORG_NAME"),
 		apiToken: os.Getenv("OKTA_API_TOKEN"),
 		domain:   os.Getenv("OKTA_BASE_URL"),
 	}
 	if err := config.loadAndValidate(); err != nil {
-		t.Fatal("Error initializing Okta client: %v", err)
+		t.Fatalf("Error initializing Okta client: %v", err)
 	}
 	return &config
 }
 
 func TestAccOktaProviderRegistration(t *testing.T) {
+	testAccPreCheck(t)
 	c := testOktaConfig(t)
 	client, err := okta.NewClientWithDomain(nil, c.orgName, c.domain, c.apiToken)
 	if err != nil {

--- a/okta/resource_policy_rules.go
+++ b/okta/resource_policy_rules.go
@@ -356,6 +356,9 @@ func policyRuleExists(d *schema.ResourceData, m interface{}) (bool, error) {
 	client := m.(*Config).oktaClient
 
 	_, _, err := client.Policies.GetPolicy(d.Get("policyid").(string))
+	if client.OktaErrorCode == "E0000007" {
+		return false, nil
+	}
 	if err != nil {
 		return false, fmt.Errorf("[ERROR] Error Listing Policy in Okta: %v", err)
 	}

